e2e/src/info.e2e-spec.ts

```diff
@@ -0,0 +1,35 @@
+import { AppPage } from './app.po';
+import { browser, by, element, logging } from 'protractor';
+
+describe('Info', () => {
+  let page: AppPage;
+
+  beforeEach(() => {
+    page = new AppPage();
+  });
+
+  it(
+    'should open the info page from the menu\n' +
+    'should display the credits information',
+  async () => {
+    await element(by.css('.btn-floating.btn-large.red')).click();
+    await element(by.css('.btn-floating.blue')).click();
+
+    expect(
+      await  element(by.css('.modal-content')).element(by.tagName('h4')).getText()
```

Comment:
Double space in here

```suggestion
      await element(by.css('.modal-content')).element(by.tagName('h4')).getText()
```